dll/np/nfs/precomp.h

```diff
@@ -0,0 +1,7 @@
+#ifdef _NFS_PRECOMP_H_
+#define _NFS_PRECOMP_H_
+
+#include <windows.h>
```

Comment:
Can we get away with WIN32_LEAN_AND_MEAN in this module?

Reply:
My rationale here was: the source files are already including the whole windows.h so the PCH should at least speed that up. It would be cool if we could get the source files improved in terms of headers use though.